faculty_cli/cli.py

```diff
@@ -558,16 +575,36 @@ def list_servers(project, all, verbose):
                 server.created_at.strftime("%Y-%m-%d %H:%M"),
             )
         )
-    if not found_servers and verbose:
+
+    if not found_servers:
```

Comment:
The existing behaviour deliberately only shows "No servers" in the `verbose` case, so that if you have a script consuming this output, it doesn't receive what looks like a single server called "No servers".

I think to keep existing behaviour we should have:

* For the old logic `not found_servers and verbose ` printing "No servers" - add as a nested `if not found_servers` under `if format == "table"`
* For the old logic `project and verbose ` printing a table with the first (project) column removed - as you've already implemented
* For the old logic `project or not verbose` printing a list of server names - as you've implemented under `else:` (but without the short circuit this comment is attached to - no servers should result in empty output)
* For the old logic `not project and verbose` printing a table printing a table including the first (project) column - as you've already implemented

Does this sound ok? Removing this short circuit would also result in the `ssh-config` format being valid in the case there are no matching servers.